### PR TITLE
bank account token for stripe integration

### PR DESCRIPTION
**Pull request by jsun1**
[Do not merge yet] Draft for adding Stripe support. Currently this is draft code for getting the bank account token from Stripe.

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -199,6 +199,31 @@ app.post('/api/set_access_token', function (request, response, next) {
...
+  ACCESS_TOKEN = request.body.access_token;
+  ACCOUNT_ID = request.body.account_id;
+  Promise.resolve()
+    .then(async function () {
+      // Generate a bank account token
+      const request = { // ProcessorStripeBankAccountTokenCreateRequest
+        access_token: ACCESS_TOKEN,
+        account_id: ACCOUNT_ID,
+      };
+      const stripeTokenResponse = await client.processorStripeBankAccountTokenCreate(
```
Comment (jsun1) [inline node/index.js:212]: I'm currently getting an undefined response here. Will investigate.

Comment (samisbakedham) [reply node/index.js:212]: Are you reading the stripe treasury docs?

Comment (jsun1) [reply node/index.js:212]: Not yet, I'm reading the Plaid+Stripe documentation: https://plaid.com/docs/auth/partnerships/stripe/

Btw I fixed this issue so now I'll merge this pull request

**Comment by jsun1** [comment]
cc @kiranmuddam

**Pull request merged**